rootfs/usr/local/bin/spacelift-tf-workspace

```diff
@@ -3,13 +3,13 @@
 # Add -x for troubleshooting
 set -ex -o pipefail
 
-terraform init -reconfigure
+tofu init -reconfigure
```

Comment:
I think this line is unnecessary and causes us to init twice which could be time consuming on large components

Reply:
I don't know if we can select the workspace without the `init`. That is something to possibly. Want to push up a build of the container that is tagged as `testing` and try it out with a single Stack?

Reply:
doesn't seem to make any difference, I guess Terraform is smart enough to `init` before selecting a workspace. In that case I'll remove it.